src/index.html

```diff
@@ -320,21 +320,23 @@ <h5>Padding</h5>
 
                     <!-- settings-link -->
                     <div id="settings-link" class="row g-1 mt-1 mb-3">
-                      <div class="mt-2 col-md-8 col-md-8">
+                      <div class="mt-2 col-md-8">                        
                         <div class="input-group">
-                          <label for="savedSettingsLink" class="input-group-text">Settings link</label>
+                          <label for="savedSettingsLink" class="input-group-text">
+                            <button id="copySettingsLink"
```

Comment:
Is there an accessibility reason for this id? If not, it doesn't have to be here, because we're not attaching any code that requires an id.